File: cses/introductoryProblems/twoKnights.cpp
#include <iostream>
using namespace std;

long long answer(long long n)
{
	return ((n * n) * (n * n - 1)/2) - (2 * (n - 2) * (n - 1) + 2 * (n - 1) * (n - 2));
}

int main(void)
{
	long long n;
	cin >> n;
	long long k = 1;
	while (k <= n)
	{
		cout << answer(k) << "\n";
		k++;
	}
	return 0;
}

File: cses/introductoryProblems/twoKnights.java
import java.util.Scanner;

public class twoKnights
{

	public static long answer(long k)
	{
		return ((k * k * (k * k - 1)/2) - 2 * (k - 2) * (k - 1) - 2 * (k - 1) * (k - 2));
	}
	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);

		long n = sc.nextLong();
		long k = 1;
		while (k <= n)
		{
			System.out.println(answer(k));
			k++;
		}
	}
}